### ISSUE TITLE
Foreground-Aware Relation Network for Geospatial Object Segmentation in High Spatial Resolution Remote Sensing Imagery 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#34]
# TL;DR
A study of segmentation task of high-resolution images. Creating a heat map of the relationship between most abstracted latent representation and feature maps of multiple resolutions, so that the model takes into account the context of the image. They also propose a mechanism to prioritize difficult samples for training.
￼
![image](https://user-images.githubusercontent.com/24524018/89614170-d9578980-d8be-11ea-8ddb-428cf01c4032.png)
![image](https://user-images.githubusercontent.com/24524018/89614175-dc527a00-d8be-11ea-87e8-a65869607dc1.png)

# Why it matters:

# Paper URL
https://openaccess.thecvf.com/content_CVPR_2020/papers/Zheng_Foreground-Aware_Relation_Network_for_Geospatial_Object_Segmentation_in_High_Spatial_CVPR_2020_paper.pdf

# Submission Dates(yyyy/mm/dd)

# Authors and institutions
Zhuo Zheng Yanfei Zhong∗ Junjue Wang Ailong Ma 
- Wuhan University, Wuhan, China 

# Methods

# Results

# Comments
CVPR2020
